sdk/deviceupdate/azure-iot-deviceupdate/pom.xml

```diff
@@ -0,0 +1,92 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-iot-deviceupdate</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-iot-deviceupdate;current} -->
+
+  <name>Microsoft Device Update for IoT Hub client library for Java</name>
+  <description>This package contains Microsoft Device Update for IoT Hub client library.</description>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.9.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
These versions seem to be pointing to old releases. Update the versions for all dependencies.

Reply:
I honestly don't know how to to do that? Where do I know the latest java versions?